subsys/nrf_security/src/drivers/cracen/cracenpsa/src/sign.c

```diff
@@ -384,8 +294,8 @@ static psa_status_t cracen_signature_ecc_sign(bool is_message,
 	if (IS_ENABLED(PSA_NEED_CRACEN_ECDSA_SECP_R1_256) &&
 	    PSA_KEY_LIFETIME_GET_LOCATION(psa_get_key_lifetime(attributes)) ==
 		    PSA_KEY_LOCATION_CRACEN) {
-		return si_ikg_sign(is_message, attributes, key_buffer, key_buffer_size, alg, input,
```

Comment:
Again not very relevant to the PR, so if you don't want to do the change let me know and I will create a task for it. But  the signature_size seem to get checked for the appropriate size later in the code. So here there is a potential memory corruption issue here.

Reply:
Oh yeah for sure that should be before. Or actually added in the validation function